lib/jekyll-import/importers/wordpressdotcom.rb

```diff
@@ -191,7 +192,14 @@ def self.process(options)
             content = Hpricot(item.text_for("content:encoded"))
             header["excerpt"] = item.excerpt if item.excerpt
 
-            download_images(item.title, content, assets_folder) if fetch
+            # Put the images into a /yyyy/mm/ subfolder to reduce clashes
+            assets_sub_folder = assets_folder
+            if item.published_at
+              date_folder = item.published_at.strftime("/%Y/%m")
+              assets_sub_folder = File.join(assets_folder, date_folder)
+            end
+
+            download_images(item.title, content, assets_sub_folder) if fetch
```

Comment:
`date_folder` isn't used elsewhere so we can avoid creating it. But then again, `assets_sub_folder` will be used only `if fetch`. So wrap the whole thing:
```suggestion
            if fetch
              # Put the images into a /yyyy/mm/ subfolder to reduce clashes
              assets_dir_path = if item.published_at
                                  File.join(assets_folder, item.published_at.strftime("/%Y/%m"))
                                else
                                  assets_folder
                                end

              download_images(item.title, content, assets_dir_path)
            end
```